ui/ui-components/client/client.ts

```diff
@@ -34,7 +40,7 @@ export class ClientCache {
     });
   }
 
-  async get(key: string) {
+  async get<T>(key: string): Promise<ClientCacheGetObject<T>> {
```

Comment:
```suggestion
  async get<T = unknown>(key: string): Promise<ClientCacheGetObject<T>> {
```

Reply:
Requires we change the return to `return { cacheData: this.cache[key]?.data as T, unlock };`

Not sure these changes get us a _ton_ since this is all behind the scenes though.